### PR TITLE
[#1102] Chore rename interfaces to match style guide

### PR DESCRIPTION
**Pull request by eonarheim**
===:clipboard: PR Checklist :clipboard:===
- [x] :pushpin: issue exists in github for these changes 
- [x] :microscope: existing tests still pass
- [x] :see_no_evil: code conforms to the [style guide](https://github.com/excaliburjs/Excalibur/blob/chore/1102-styleguide/STYLEGUIDE.md)
- [x] :triangular_ruler: new tests written and passing / old tests updated with new scenario(s)
- [x] :page_facing_up: changelog entry added (or not needed)

==================

Rename interfaces to match styleguide #1102

### CODE REVIEW

File: GruntFile.js [hunk 1]
```diff
@@ -295,6 +295,9 @@ module.exports = function(grunt) {
   // Default task - compile & test
   grunt.registerTask('default', ['tslint:src', 'core', 'karma', 'visual']);
 
+  // Lint only
+  grunt.registerTask('tslint', ['tslint:src']);
```
Comment (kamranayub) [inline GruntFile.js]: Thoughts on simply `lint`?

**Comment by kamranayub** [comment]
I see you added aliases for deprecation (yes!), do those work as expected when pulled into a project do you know?

File: GruntFile.js (hunk 1 above)
Comment (eonarheim) [reply GruntFile.js]: lint works

**Comment by eonarheim** [comment]
@kamranayub sandbox behaves as expected

image

**Pull request merged**